1.0.0: pytest is failing in 3 units

I'm trying to package your module as an rpm package. So I'm using the typical PEP517 based build, install and test cycle used on building packages from non-root account.
- `python3 -sBm build -w --no-isolation`
- because I'm calling `build` with `--no-isolation` I'm using during all processes oly locally installed modules
- install .whl file in </install/prefix>
- run pytest with PYTHONPATH pointing to sitearch and sitelib inside </install/prefix>

Here is pytest output:
```console
+ PYTHONPATH=/home/tkloczko/rpmbuild/BUILDROOT/python-sphinx-hoverxref-1.0.0-2.fc35.x86_64/usr/lib64/python3.8/site-packages:/home/tkloczko/rpmbuild/BUILDROOT/python-sphinx-hoverxref-1.0.0-2.fc35.x86_64/usr/lib/python3.8/site-packages
+ /usr/bin/pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.12, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/tkloczko/rpmbuild/BUILD/sphinx-hoverxref-1.0.0, configfile: pytest.ini
collected 13 items

tests/test_htmltag.py .........FFF                                                                                                                                   [ 92%]
tests/test_internals.py .                                                                                                                                            [100%]

================================================================================= FAILURES =================================================================================
_____________________________________________________________________ test_intersphinx_default_configs _____________________________________________________________________

app = <SphinxTestApp buildername='html'>, status = <_io.StringIO object at 0x7f43838bd8b0>, warning = <_io.StringIO object at 0x7f43838bddc0>

    @pytest.mark.sphinx(
        srcdir=intersphinxsrc,
    )
    def test_intersphinx_default_configs(app, status, warning):
        app.build()
        path = app.outdir / 'index.html'
        assert path.exists() is True
        content = open(path).read()

        chunks_regex = [
            r'<a class="reference external" href="https://docs.python.org/3/tutorial/index.html#tutorial-index" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to The Python Tutorial using intersphinx</span></a>',
            r'<a class="reference external" href="https://docs.python.org/3/library/datetime.html#datetime-datetime" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to datetime.datetime Python’s function using intersphinx</span></a>',
            r'<a class="reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs v\d\d?.\d\d?.\d\d?\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',
            r'<a class="reference external" href="https://docs.python.org/3/library/functions.html#float" title="\(in Python v3.\d\d?\)"><code class="xref py py-class docutils literal notranslate"><span class="pre">float</span></code></a>',
        ]

        chunks = [
            '<a class="reference internal" href="#hoverxref.extension.setup" title="hoverxref.extension.setup"><code class="xref py py-func docutils literal notranslate"><span class="pre">hoverxref.extension.setup()</span></code></a>',
        ]

        if sphinx.version_info >= (4, 0):
            chunks.extend([
                '<dt class="sig sig-object py" id="hoverxref.extension.setup">',
            ])
        else:
            chunks.extend([
                '<dt id="hoverxref.extension.setup">',
            ])


        for chunk in chunks:
            assert chunk in content

        for chunk in chunks_regex:
>           assert re.search(chunk, content)
E           assert None
E            +  where None = <function search at 0x7f4386a85a60>('<a class="reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\\(in Re...\d?\\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>', '\n<!DOCTYPE html>\n\n<html>\n  <head>\n    <meta charset="utf-8" />\n    <meta name="viewport" content="width=device-...ref="_sources/index.rst.txt"\n          rel="nofollow">Page source</a>\n    </div>\n\n    \n\n    \n  </body>\n</html>')
E            +    where <function search at 0x7f4386a85a60> = re.search

tests/test_htmltag.py:255: AssertionError
------------------------------------------------------------------------- Captured stdout teardown -------------------------------------------------------------------------
# testroot: root
# builder: html
# srcdir: /home/tkloczko/rpmbuild/BUILD/sphinx-hoverxref-1.0.0/tests/examples/intersphinx
# outdir: /home/tkloczko/rpmbuild/BUILD/sphinx-hoverxref-1.0.0/tests/examples/intersphinx/_build/html
# status:
Running Sphinx v4.3.2
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://docs.readthedocs.io/en/stable/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 1 source files that are out of date
updating environment: [new config] 1 added, 0 changed, 0 removed
reading sources... [100%] index
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index
generating indices... genindex done
writing additional pages... search done
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in tests/examples/intersphinx/_build/html.

# warning:

_____________________________________________________________________ test_intersphinx_python_mapping ______________________________________________________________________

app = <SphinxTestApp buildername='html'>, status = <_io.StringIO object at 0x7f43822b3670>, warning = <_io.StringIO object at 0x7f43822b3430>

    @pytest.mark.sphinx(
        srcdir=intersphinxsrc,
        confoverrides={
            'hoverxref_intersphinx': [
                'python',
            ],
        },
    )
    def test_intersphinx_python_mapping(app, status, warning):
        app.build()
        path = app.outdir / 'index.html'
        assert path.exists() is True
        content = open(path).read()

        chunks_regex = [
            # Python's links do have hoverxref enabled
            r'<a class="hoverxref tooltip reference external" href="https://docs.python.org/3/tutorial/index.html#tutorial-index" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to The Python Tutorial using intersphinx</span></a>',
            r'<a class="hoverxref tooltip reference external" href="https://docs.python.org/3/library/datetime.html#datetime-datetime" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to datetime.datetime Python’s function using intersphinx</span></a>',
            r'<a class="hoverxref tooltip reference external" href="https://docs.python.org/3/library/functions.html#float" title="\(in Python v3.\d\d?\)"><code class="xref py py-class docutils literal notranslate"><span class="pre">float</span></code></a>',

            # Read the Docs' link does not have hoverxref enabled
            r'<a class="reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs v\d\d?.\d\d?.\d\d?\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',

        ]

        chunks = [
            # Python's domain does not have hoverxref enabled
            '<a class="reference internal" href="#hoverxref.extension.setup" title="hoverxref.extension.setup"><code class="xref py py-func docutils literal notranslate"><span class="pre">hoverxref.extension.setup()</span></code></a>',
        ]

        for chunk in chunks:
            assert chunk in content

        for chunk in chunks_regex:
>           assert re.search(chunk, content)
E           assert None
E            +  where None = <function search at 0x7f4386a85a60>('<a class="reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\\(in Re...\d?\\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>', '\n<!DOCTYPE html>\n\n<html>\n  <head>\n    <meta charset="utf-8" />\n    <meta name="viewport" content="width=device-...ref="_sources/index.rst.txt"\n          rel="nofollow">Page source</a>\n    </div>\n\n    \n\n    \n  </body>\n</html>')
E            +    where <function search at 0x7f4386a85a60> = re.search

tests/test_htmltag.py:292: AssertionError
------------------------------------------------------------------------- Captured stdout teardown -------------------------------------------------------------------------
# testroot: root
# builder: html
# srcdir: /home/tkloczko/rpmbuild/BUILD/sphinx-hoverxref-1.0.0/tests/examples/intersphinx
# outdir: /home/tkloczko/rpmbuild/BUILD/sphinx-hoverxref-1.0.0/tests/examples/intersphinx/_build/html
# status:
Running Sphinx v4.3.2
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://docs.readthedocs.io/en/stable/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 1 source files that are out of date
updating environment: [new config] 1 added, 0 changed, 0 removed
reading sources... [100%] index
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index
generating indices... genindex done
writing additional pages... search done
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in tests/examples/intersphinx/_build/html.

# warning:

______________________________________________________________________ test_intersphinx_all_mappings _______________________________________________________________________

app = <SphinxTestApp buildername='html'>, status = <_io.StringIO object at 0x7f4381f25ca0>, warning = <_io.StringIO object at 0x7f4381f259d0>

    @pytest.mark.sphinx(
        srcdir=intersphinxsrc,
        confoverrides={
            'hoverxref_intersphinx': [
                'readthedocs',
                'python',
            ],
            'hoverxref_intersphinx_types': {
                'readthedocs': 'modal',
                'python': {
                    'class': 'modal',
                }
            },
            'hoverxref_domains': ['py'],
        },
    )
    def test_intersphinx_all_mappings(app, status, warning):
        app.build()
        path = app.outdir / 'index.html'
        assert path.exists() is True
        content = open(path).read()

        chunks_regex = [
            # Python's links do have hoverxref enabled
            r'<a class="hoverxref tooltip reference external" href="https://docs.python.org/3/tutorial/index.html#tutorial-index" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to The Python Tutorial using intersphinx</span></a>',
            r'<a class="hoverxref tooltip reference external" href="https://docs.python.org/3/library/datetime.html#datetime-datetime" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to datetime.datetime Python’s function using intersphinx</span></a>',
            r'<a class="hoverxref modal reference external" href="https://docs.python.org/3/library/functions.html#float" title="\(in Python v3.\d\d?\)"><code class="xref py py-class docutils literal notranslate"><span class="pre">float</span></code></a>',

            # Read the Docs' link does have hoverxref enabled
            r'<a class="hoverxref modal reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs v\d\d?.\d\d?.\d+\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',

        ]

        chunks = [
            # Python domain's link does have hoverxref enabled
            '<a class="hoverxref tooltip reference internal" href="#hoverxref.extension.setup" title="hoverxref.extension.setup"><code class="xref py py-func docutils literal notranslate"><span class="pre">hoverxref.extension.setup()</span></code></a>',
        ]

        for chunk in chunks:
            assert chunk in content

        for chunk in chunks_regex:
>           assert re.search(chunk, content)
E           assert None
E            +  where None = <function search at 0x7f4386a85a60>('<a class="hoverxref modal reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python"...\d+\\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>', '\n<!DOCTYPE html>\n\n<html>\n  <head>\n    <meta charset="utf-8" />\n    <meta name="viewport" content="width=device-...ref="_sources/index.rst.txt"\n          rel="nofollow">Page source</a>\n    </div>\n\n    \n\n    \n  </body>\n</html>')
E            +    where <function search at 0x7f4386a85a60> = re.search

tests/test_htmltag.py:337: AssertionError
------------------------------------------------------------------------- Captured stdout teardown -------------------------------------------------------------------------
# testroot: root
# builder: html
# srcdir: /home/tkloczko/rpmbuild/BUILD/sphinx-hoverxref-1.0.0/tests/examples/intersphinx
# outdir: /home/tkloczko/rpmbuild/BUILD/sphinx-hoverxref-1.0.0/tests/examples/intersphinx/_build/html
# status:
Running Sphinx v4.3.2
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://docs.readthedocs.io/en/stable/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 1 source files that are out of date
updating environment: [new config] 1 added, 0 changed, 0 removed
reading sources... [100%] index
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
Using default style (tooltip) for unknown typ (func). Define it in hoverxref_role_types.

generating indices... genindex done
writing additional pages... search done
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in tests/examples/intersphinx/_build/html.

# warning:

========================================================================= short test summary info ==========================================================================
FAILED tests/test_htmltag.py::test_intersphinx_default_configs - assert None
FAILED tests/test_htmltag.py::test_intersphinx_python_mapping - assert None
FAILED tests/test_htmltag.py::test_intersphinx_all_mappings - assert None
======================================================================= 3 failed, 10 passed in 7.48s =======================================================================
```
List of installed modules in build env:
```console
Package                       Version
----------------------------- -------------------
alabaster                     0.7.12
appdirs                       1.4.4
attrs                         21.4.0
Babel                         2.9.1
build                         0.7.0
chardet                       4.0.0
charset-normalizer            2.0.10
cycler                        0.11.0
dbus-python                   1.2.18
distro                        1.6.0
docutils                      0.17.1
extras                        1.0.0
fixtures                      3.0.0
fonttools                     4.28.5
gpg                           1.16.0-unknown
idna                          3.3
imagesize                     1.3.0
importlib-metadata            4.10.0
iniconfig                     1.1.1
Jinja2                        3.0.3
kiwisolver                    1.3.2
latexcodec                    2.0.1
libcomps                      0.1.18
lit                           13.0.0
lxml                          4.7.1
Mako                          1.1.6.dev0
Markdown                      3.3.6
MarkupSafe                    2.0.1
matplotlib                    3.5.1
meson                         0.60.3
numpy                         1.22.0
olefile                       0.46
packaging                     21.3
pbr                           5.8.0
pep517                        0.12.0
Pillow                        9.0.0
pip                           21.3.1
pluggy                        1.0.0
py                            1.11.0
pybtex                        0.24.0
pybtex-docutils               1.0.1
Pygments                      2.11.2
PyGObject                     3.42.0
pyparsing                     3.0.6
pytest                        6.2.5
python-dateutil               2.8.2
pytz                          2021.1
PyYAML                        6.0
requests                      2.27.1
rpm                           4.17.0
scour                         0.38.2
setuptools                    60.1.0.post20211225
six                           1.16.0
smartypants                   2.0.1
snowballstemmer               2.2.0
Sphinx                        4.3.2.dev20220106
sphinx-prompt                 1.4.0
sphinx-tabs                   3.2.0
sphinxcontrib-applehelp       1.0.2.dev20220108
sphinxcontrib-bibtex          2.4.1
sphinxcontrib-devhelp         1.0.2.dev20220108
sphinxcontrib-htmlhelp        2.0.0
sphinxcontrib-jsmath          1.0.1.dev20220108
sphinxcontrib-qthelp          1.0.3.dev20220108
sphinxcontrib-serializinghtml 1.1.5
testtools                     2.5.0
toml                          0.10.2
tomli                         2.0.0
typogrify                     2.0.7
urllib3                       1.26.7
wheel                         0.37.1
zipp                          3.7.0
```

--- a/tests/test_htmltag.py
+++ b/tests/test_htmltag.py
@@ -230,7 +230,7 @@ def test_intersphinx_default_configs(app, status, warning):
     chunks_regex = [
         r'<a class="reference external" href="https://docs.python.org/3/tutorial/index.html#tutorial-index" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to The Python Tutorial using intersphinx</span></a>',
         r'<a class="reference external" href="https://docs.python.org/3/library/datetime.html#datetime-datetime" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to datetime.datetime Python’s function using intersphinx</span></a>',
-        r'<a class="reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs v\d\d?.\d\d?.\d\d?\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',
+        r'<a class="reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs user documentation v\d\d?.\d\d?.\d\d?\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',
         r'<a class="reference external" href="https://docs.python.org/3/library/functions.html#float" title="\(in Python v3.\d\d?\)"><code class="xref py py-class docutils literal notranslate"><span class="pre">float</span></code></a>',
     ]
 
@@ -276,7 +276,7 @@ def test_intersphinx_python_mapping(app, status, warning):
         r'<a class="hoverxref tooltip reference external" href="https://docs.python.org/3/library/functions.html#float" title="\(in Python v3.\d\d?\)"><code class="xref py py-class docutils literal notranslate"><span class="pre">float</span></code></a>',
 
         # Read the Docs' link does not have hoverxref enabled
-        r'<a class="reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs v\d\d?.\d\d?.\d\d?\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',
+        r'<a class="reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs user documentation v\d\d?.\d\d?.\d\d?\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',
 
     ]
 
@@ -321,7 +321,7 @@ def test_intersphinx_all_mappings(app, status, warning):
         r'<a class="hoverxref modal reference external" href="https://docs.python.org/3/library/functions.html#float" title="\(in Python v3.\d\d?\)"><code class="xref py py-class docutils literal notranslate"><span class="pre">float</span></code></a>',
 
         # Read the Docs' link does have hoverxref enabled
-        r'<a class="hoverxref modal reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs v\d\d?.\d\d?.\d+\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',
+        r'<a class="hoverxref modal reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs user documentation v\d\d?.\d\d?.\d+\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',
 
     ]
 

